checkout-v3/enterprise/redirect.md

```diff
@@ -94,6 +94,6 @@ capture and the other options you have after the purchase.
                          next_href="post-purchase"
                          next_title="Post Purchase" %}
 
-[abort-feature]: /checkout-v3/mac/features/core/abort
-[sequence-diagram]: /checkout-v3/sequence-diagrams/#merchant-authenticated-consumer-redirect
-[redirect-mac-menu]: /assets/img/checkout/mac-redirect.png
+[abort-feature]: /checkout-v3/enterprise/features/core/abort
+[sequence-diagram]: /checkout-v3/sequence-diagrams/#enterprise-redirect
+[redirect-enterprise-menu]: /assets/img/checkout/enterprise-redirect.png
```

Comment:
Has the actual images changed name too? We should replace with new screenshots

Reply:
I agree the screenshots should be replaced, but let's do that in a separate task so this can be merged?

Reply:
Agreed!